api/api/Controllers/ApiController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Data.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace api.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api")]
+
+    public class ApiController : Controller
+    {
+        [HttpGet("getVersion")]
+        public async Task<ActionResult> GetVersion()
+        {
+            var versionInfo = new
+            {
+                version = "1.0.0",
+                timestamp = DateTime.UtcNow.ToString("o")
+            };
```

Comment:
this should be stored on a json file, so you can call the version from there

Reply:
added in another PR because of git ignore conflicts